### PR TITLE
expose both dash and underscore variants

### PR DESCRIPTION
**Pull request by timkpaine**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by timkpaine** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,6 +32,18 @@ test:
   imports:
     - jupyter_packaging
 
+
+outputs:
+  - name: jupyter-packaging
+  - name: jupyter_packaging
```
Comment (timkpaine) [inline recipe/meta.yaml:38]: @conda-forge/jupyter-packaging lmk if there are any objections to exposing this package as 2 `conda`-installable packages, `jupyter-packaging` and `jupyter_packaging`

Comment (blink1073) [reply recipe/meta.yaml:38]: Hey @timkpaine, I think having two packages would cause some confusion, why not stick with the same package name as pypi?

Comment (timkpaine) [reply recipe/meta.yaml:38]: Pypi supports dash and underscore as equivalent, so this actually makes the conda behavior the same as pypi

Comment (blink1073) [reply recipe/meta.yaml:38]: Fair enough, I don't feel strongly either way

Comment (xmnlab) [reply recipe/meta.yaml:38]: no objections

**Pull request merged**